contracts/JBController.sol

```diff
@@ -542,16 +578,16 @@ contract JBController is IJBController, JBTerminalUtility, JBOperatable, Reentra
     )
   {
     // There should be tokens to burn
-    require(_tokenCount > 0, '0x32: NO_OP');
-
+    if (_tokenCount == 0) {
+      revert ZERO_TOKEN_TO_BURN();
+    }
     // Get a reference to the project's current funding cycle.
     JBFundingCycle memory _fundingCycle = fundingCycleStore.currentOf(_projectId);
 
     // If the message sender is not a terminal delegate, the current funding cycle must not be paused.
-    require(
-      !_fundingCycle.burnPaused() || directory.isTerminalDelegateOf(_projectId, msg.sender),
-      '0x33: PAUSED'
-    );
+    if (_fundingCycle.burnPaused() && !directory.isTerminalDelegateOf(_projectId, msg.sender)) {
+      revert BURN_PAUSED();
+    }
```

Comment:
Same here, error should be match the condition more accurately.

Reply:
This one is ok no? If burn is paused, a terminal delegate can still burn